library/include/ck/library/reference_tensor_operation/cpu/reference_maxpool_bwd.hpp

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <iostream>
+#include <sstream>
+#include <vector>
+#include <algorithm>
```

Comment:
Remove including of `algorithm`,  which I think is not used

Reply:
ed4912f2f6a8d6aba6404ebb46b75ed1b39979cd